components/Slider/tests/unit/SliderTests.m

```diff
@@ -1103,6 +1107,70 @@ - (void)testAccessibilityIncrementDiscreteSlider {
   }
 }
 
+- (void)testDefaultHapticsEnabledValue {
+  if (@available(iOS 10.0, *)) {
+    XCTAssertTrue(self.slider.hapticsEnabled);
+  } else {
+    XCTAssertFalse(self.slider.hapticsEnabled);
+  }
+}
+
+- (void)testEnabledHapticFeedback {
+  //Given
+  self.slider.minimumValue = 0;
+  self.slider.maximumValue = 10;
+  self.slider.hapticsEnabled = YES;
+
+
+  if (@available(iOS 10.0, *)) {
+    for (NSUInteger i = 0; i < 11; ++i) {
+        self.slider.value = i;
+        _mockFeedbackGenerator = [[MockUIImpactFeedbackGenerator alloc] init];
+        self.slider.feedbackGenerator = _mockFeedbackGenerator;
```

Comment:
can these 2 lines be moved to outside the for loop so it is instantiated only one